test/conftest.py

```diff
@@ -77,7 +77,7 @@ def insert_data_from_csv(db, request):
     used to parse semantics of the .csv file
     """
     def insert_data_from_csv(in_csv_file, rec_count=-1, expected_error_count=0, loader_result_lambda=None):
-        if rec_count > 0 and rec_count < 80:
+        if rec_count > 0 and rec_count < 80 or "descriptorset" in in_csv_file:
             request.param = False
             print("Not enough records to test parallel loader. Using serial loader.")
```

Comment:
seems like having descriptorset will also cause the error message as well, should add that to the print?

Reply:
The reason is still the same. The CSV we have is tiny one. 
Even then This print in tests is hardly of use, and if we still try to use tiny files in dataCSV, they get [caught](https://github.com/aperture-data/aperturedb-python/blob/233f9397a64d651c6011a2df3043f2ef881b5db2/aperturedb/CSVParser.py#L70).

Reply:
I'm going to mark this resolved for now.